src/lib/peer_trust/peer_trust.mli

```diff
@@ -8,6 +8,8 @@
     caller, which is coda_base. *)
 
 open Core
+open Async
```

Comment:
Should the opening comment mention that the initial value of trust is 0 ?

Reply:
yeah